src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components'
+import IconHome from '../../components/assets/icons/IconHome.svg'
+import IconPlus from '../../components/assets/icons/IconPlus.svg'
+import IconMonthlyOverview from '../../components/assets/icons/IconMonthlyOverview.svg'
+
+export default function Navigation() {
+  return (
+    <NavigationStyled>
+      <NavLink exact to="/">
+        <img src={IconHome} alt="Home" />
+      </NavLink>
+      <NavLink to="/Create">
+        <AddIcon src={IconPlus} alt="Create" />
+      </NavLink>
+      <NavLink to="/Overview">
+        <img src={IconMonthlyOverview} alt="Monthly Overview" />
```

Comment:
Your alt attributes are a little imprecise - If a user, who needs a screen-reader will hear the description "home" I guess it is not very helpful. Maybe you can precise it a bit more or even remove it if it has no value for such users.